client/src/components/new-request/grey-box.vue

```diff
@@ -450,6 +294,28 @@ export default class GreyBox extends Vue {
     }
     return []
   }
+  get optionClasses () {
```

Comment:
This was previously implemented as a method.  as the i parameter is actually the same `this.i`, have moved it to be a getter.